supertokens_python/recipe/session/access_token.py

```diff
@@ -45,45 +45,29 @@ def sanitize_number(n: Any) -> Union[Union[int, float], None]:
 
 def get_info_from_access_token(
     jwt_info: ParsedJWTInfo,
-    jwk_clients: List[JWKClient],
+    combined_jwks: List[PyJWK],
     do_anti_csrf_check: bool,
 ):
     try:
         payload: Optional[Dict[str, Any]] = None
-        client: Optional[JWKClient] = None
-        keys: Optional[List[PyJWK]] = None
-
-        # Get the keys from the first available client
-        for c in jwk_clients:
-            try:
-                client = c
-                keys = c.get_latest_keys()
-                break
-            except JWKSRequestError:
-                continue
-
-        if keys is None or client is None:
-            raise PyJWKClientError("No key found")
-
-        if jwt_info.version < 3:
+        if jwt_info.version >= 3:
+            matching_key = [k for k in combined_jwks if k.key_id == jwt_info.kid][0]  # type: ignore
+            payload = jwt.decode(  # type: ignore
+                jwt_info.raw_token_string,
+                matching_key.key,  # type: ignore
+                algorithms=["RS256"],
```

Comment:
Why hardcode the algorithms like this?

Reply:
If I don't it throws: `It is required that you pass in a value for the "algorithms" argument when calling decode().`

Reply:
No I mean why hardcode it? What happens if we have more algorithms for the access token in the future? We would specifically need to change the python SDK only for it to work. You could read the algorithm from the header and pass that in instead

Reply:
Okay.

Reply:
Done